package.json

```diff
@@ -90,7 +90,8 @@
     "root": true
   },
   "resolutions": {
-    "@mui/styled-engine": "npm:@mui/styled-engine-sc@latest"
+    "@mui/styled-engine": "npm:@mui/styled-engine-sc@latest",
+    "rollup": "npm:rollup@3.29.4"
```

Comment:
Do we need this? (for watcher, the nebula issue that it does not picks changes while developing)
[PVT is currently using it](https://github.com/qlik-oss/sn-pivot-table/blob/177d3991d0e4bcbd1d68d8df596f4f405b757af0/package.json#L83-L85)